Octokit/Http/ApiConnection.cs

```diff
@@ -338,6 +338,20 @@ public Task Put(Uri uri)
             return Connection.Put(uri);
         }
 
+        /// <summary>
+        /// Creates or replaces the API resource at the specified URI
+        /// </summary>
+        /// <param name="uri">URI of the API resource to put</param>
+        /// <param name="data">Object that describes the API resource; this will be serialized and used as the request's body</param>
+        /// <returns>A <see cref="Task"/> for the request's execution.</returns>
+        public Task Put(Uri uri, object data)
```

Comment:
Do you mind ELI5ing the reason for this helper?

Reply:
I needed a non-generic Put method where I could pass data for the request's body for AddOrUpdateTeamRepositoryPermissions method on TeamsClients, which returns a 204.